config/projects/datadog-agent.rb

```diff
@@ -193,7 +193,7 @@
 dependency 'psutil'
 dependency 'requests'
 
-if !ENV["USE_INTEGRATION_SDK"]
+unless ENV["USE_INTEGRATION_SDK"] == 'true'
```

Comment:
Yeah I think this is probably better than checking for nil... The `truthy` stuff is tricky in ruby.